src/domain/LevelState.ts

```diff
@@ -0,0 +1,12 @@
+import { reactive } from 'vue'
+import type { Level } from './Level'
+
+export interface LevelState {
+  currentLevel: Level | null
```

Comment:
```suggestion
  currentLevel: Level
```

There is no possibility currentLevel is null